Source/Tests/ELFCodeLoader2.h

```diff
@@ -472,9 +473,18 @@ class ELFCodeLoader2 final : public FEXCore::CodeLoader {
       if (!NoRandomize) {
         constexpr uint64_t ASLR_BITS_64 = 28;
         constexpr uint64_t ASLR_BITS_32 = 8;
-        std::random_device rd;
-        std::uniform_int_distribution<uint64_t> d(0);
-        uint64_t ASLR_Offset = d(rd);
+        uint64_t ASLR_Offset{};
+        ssize_t Result{};
+        do {
+          // Use the getrandom syscall instead of std::random_device because std can assert.
+          // <256 bytes of RNG requested, guaranteed to not be interrupted by signal.
```

Comment:
Are these two unrelated facts, or does one imply the other? In the latter case I'd suggest:
```suggestion
          // This is guaranteed to not be interrupted by a signal,
          // since fewer than 256 bytes of RNG data are requested
```

Reply:
Modified comment.